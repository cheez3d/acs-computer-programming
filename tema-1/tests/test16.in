rosu galben albastru verde
8884446665553380624336682055255444
rosu este o culoare calda
2
calda rece
rosu albastru
roz alb maro
