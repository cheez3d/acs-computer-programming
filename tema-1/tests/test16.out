7776667777880425552233660255522#277778777880888337773#33
violet magenta kaki
255522#277778777880337777833066602228855566627773307773322233
3
7776669999 255522 62777666 
7776669999 1
0
0
